README.md

```diff
@@ -200,6 +201,8 @@ Additional properties:
 * `parent` - ID of another image in this build which is used as a parent of an image in `FROM` statement
 * `extra_files` - list of additional files and folders to be included in checksum (files and folders from the same directory where
 an image-specific `Dockerfile.template` is located are included by default)
+* `exclude_files` - list of additional files and folders to be excluded from the checksum (this can be used for e.g. ignoring
```

Comment:
nits for future improvement:

probably `_paths` is more accurate given they can be directories